pkg/apis/servicecatalog/validation/serviceclass.go

```diff
@@ -55,6 +55,10 @@ func validateExternalID(value string) []string {
 
 // ValidateServiceClass validates a ServiceClass and returns a list of errors.
 func ValidateServiceClass(serviceclass *sc.ServiceClass) field.ErrorList {
+	return internalValidateServiceClass(serviceclass)
+}
+
+func internalValidateServiceClass(serviceclass *sc.ServiceClass) field.ErrorList {
```

Comment:
Just curious, why was this split out from the func above?

Reply:
For consistency with other files in the same directory. Also there might be other validation that we might want to control differently during an update vs. create.